pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -828,19 +828,16 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *v1.Serv
 
 	status.Ingress = []v1.LoadBalancerIngress{{IP: loadbalancer.VipAddress}}
 
-	port, err := getPortByIP(lbaas.network, loadbalancer.VipAddress)
-	if err != nil {
-		return nil, fmt.Errorf("Error getting port for LB vip %s: %v", loadbalancer.VipAddress, err)
-	}
-	floatIP, err := getFloatingIPByPortID(lbaas.network, port.ID)
+	portID := loadbalancer.VipPortID
```

Comment:
The change looks good otherwise, so I'm happy to let this go ahead and let it get more widespread testing.  This section of code has been usually full of dragons, and most of it not caused by us.

/lgtm